// partial-ivy/esm2020/lib/partial-ivy.module.mjs
import { NgModule } from '@angular/core';
import { PartialIvyComponent } from './partial-ivy.component';
import * as i0 from "@angular/core";
export class PartialIvyModule {
}
PartialIvyModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
PartialIvyModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyModule, declarations: [PartialIvyComponent], exports: [PartialIvyComponent] });
PartialIvyModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyModule, imports: [[]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        PartialIvyComponent
                    ],
                    imports: [],
                    exports: [
                        PartialIvyComponent
                    ]
                }]
        }] });
