// partial-ivy/esm2020/lib/partial-ivy.component.mjs
import { Component } from '@angular/core';
import * as i0 from "@angular/core";
export class PartialIvyComponent {
    constructor() { }
    ngOnInit() {
    }
}
PartialIvyComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
PartialIvyComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "12.0.0", version: "13.0.2", type: PartialIvyComponent, selector: "lib-partial-ivy", ngImport: i0, template: `
    <p>
      partial-ivy works!
    </p>
  `, isInline: true });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyComponent, decorators: [{
            type: Component,
            args: [{
                    selector: 'lib-partial-ivy',
                    template: `
    <p>
      partial-ivy works!
    </p>
  `,
                    styles: []
                }]
        }], ctorParameters: function () { return []; } });
