// partial-ivy/fesm2020/partial-ivy.mjs
import * as i0 from '@angular/core';
import { Injectable, Component, NgModule } from '@angular/core';

class PartialIvyService {
    constructor() { }
}
PartialIvyService.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyService, deps: [], target: i0.ɵɵFactoryTarget.Injectable });
PartialIvyService.ɵprov = i0.ɵɵngDeclareInjectable({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyService, providedIn: 'root' });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyService, decorators: [{
            type: Injectable,
            args: [{
                    providedIn: 'root'
                }]
        }], ctorParameters: function () { return []; } });

class PartialIvyComponent {
    constructor() { }
    ngOnInit() {
    }
}
PartialIvyComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
PartialIvyComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "12.0.0", version: "13.0.2", type: PartialIvyComponent, selector: "lib-partial-ivy", ngImport: i0, template: `
    <p>
      partial-ivy works!
    </p>
  `, isInline: true });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyComponent, decorators: [{
            type: Component,
            args: [{
                    selector: 'lib-partial-ivy',
                    template: `
    <p>
      partial-ivy works!
    </p>
  `,
                    styles: []
                }]
        }], ctorParameters: function () { return []; } });

class PartialIvyModule {
}
PartialIvyModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
PartialIvyModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyModule, declarations: [PartialIvyComponent], exports: [PartialIvyComponent] });
PartialIvyModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyModule, imports: [[]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.0.2", ngImport: i0, type: PartialIvyModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        PartialIvyComponent
                    ],
                    imports: [],
                    exports: [
                        PartialIvyComponent
                    ]
                }]
        }] });

/*
 * Public API Surface of partial-ivy
 */

/**
 * Generated bundle index. Do not edit.
 */

export { PartialIvyComponent, PartialIvyModule, PartialIvyService };
