// full-ivy/fesm2020/full-ivy.mjs
import * as i0 from '@angular/core';
import { Injectable, Component, NgModule } from '@angular/core';

class FullIvyService {
    constructor() { }
}
FullIvyService.ɵfac = function FullIvyService_Factory(t) { return new (t || FullIvyService)(); };
FullIvyService.ɵprov = /*@__PURE__*/ i0.ɵɵdefineInjectable({ token: FullIvyService, factory: FullIvyService.ɵfac, providedIn: 'root' });
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && i0.ɵsetClassMetadata(FullIvyService, [{
        type: Injectable,
        args: [{
                providedIn: 'root'
            }]
    }], function () { return []; }, null); })();

class FullIvyComponent {
    constructor() { }
    ngOnInit() {
    }
}
FullIvyComponent.ɵfac = function FullIvyComponent_Factory(t) { return new (t || FullIvyComponent)(); };
FullIvyComponent.ɵcmp = /*@__PURE__*/ i0.ɵɵdefineComponent({ type: FullIvyComponent, selectors: [["lib-full-ivy"]], decls: 2, vars: 0, template: function FullIvyComponent_Template(rf, ctx) { if (rf & 1) {
        i0.ɵɵelementStart(0, "p");
        i0.ɵɵtext(1, " full-ivy works! ");
        i0.ɵɵelementEnd();
    } }, encapsulation: 2 });
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && i0.ɵsetClassMetadata(FullIvyComponent, [{
        type: Component,
        args: [{
                selector: 'lib-full-ivy',
                template: `
    <p>
      full-ivy works!
    </p>
  `,
                styles: []
            }]
    }], function () { return []; }, null); })();

class FullIvyModule {
}
FullIvyModule.ɵfac = function FullIvyModule_Factory(t) { return new (t || FullIvyModule)(); };
FullIvyModule.ɵmod = /*@__PURE__*/ i0.ɵɵdefineNgModule({ type: FullIvyModule });
FullIvyModule.ɵinj = /*@__PURE__*/ i0.ɵɵdefineInjector({ imports: [[]] });
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && i0.ɵsetClassMetadata(FullIvyModule, [{
        type: NgModule,
        args: [{
                declarations: [
                    FullIvyComponent
                ],
                imports: [],
                exports: [
                    FullIvyComponent
                ]
            }]
    }], null, null); })();
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && i0.ɵɵsetNgModuleScope(FullIvyModule, { declarations: [FullIvyComponent], exports: [FullIvyComponent] }); })();

/*
 * Public API Surface of full-ivy
 */

/**
 * Generated bundle index. Do not edit.
 */

export { FullIvyComponent, FullIvyModule, FullIvyService };
